ebpfsvc/platform.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#define WIN32_LEAN_AND_MEAN // Exclude rarely-used stuff from Windows headers
+#include "pch.h"
+#include <stdint.h>
+#include "ebpf_api.h"
+
+namespace Platform {
+BOOL
```

Comment:
use standard types where they exist, not the upper case aliases

Reply:
fixed.